ocaml/xapi/vm_platform.ml

```diff
@@ -44,6 +44,10 @@ let igd_passthru_key = Xapi_globs.igd_passthru_key
 let featureset = "featureset"
 let nested_virt = "nested-virt"
 
+(* The default value of device model should be set as
```

Comment:
How does this work with the following section from the design?

```
* XAPI shall have a default device model profile. The default can be overridden
by setting `pool.other_config["device_model_default"]` to a particular profile.
```

The value defined here does not take this into account.

Reply:
Hi @lindig , thanks for your comments.
I double checked the design change, these lines are removed now.
```
-* XAPI shall have a default device model profile. The default can be overridden
-by setting `pool.other_config["device_model_default"]` to a particular profile.
-
```
 https://code.citrite.net/projects/XSC/repos/xs-documents/pull-requests/19/commits/f546a53424ca8d823b55010d26978e93c5d1bf68?since=dd4052a704de4764c1f80ee6aafdb1fb7027ec37

Reply:
This is not merged into the master branch of the design document - which is what I checked. But I understand that this is likely to change and would mean that the default is static.

Reply:
Thanks, so we will not consider `pool.other_config["device_model_default"]` now.